### ISSUE TITLE
Spamming ham in the debug menu while standing over the fire crashes the game

### ISSUE THREAD

**Issue by hardliner66** [foxnne/aftersun#8]
Basically, title. It seems when you spam ham over the fire, it somehow tries to access some entity that's already gone, which crashes the game.

Error:
```
fatal: flecs.c: 7994: assert: ecs_is_alive(world, entity) INVALID_PARAMETER
run Aftersun: error: the following command exited with error code 3:
C:\projects\zig\aftersun\zig-out\bin\Aftersun.exe
Build Summary: 11/13 steps succeeded; 1 failed (disable with --summary none)
run transitive failure
└─ run Aftersun failure
```

**Comment by foxnne** [comment]
Thanks for this report! Honestly the spawning of entities and the cooking system is probably the most questionable thing in here, I'm not too surprised you found a bug with it. I'll look into this and potentially rework the way it works altogether.

**Comment by hardliner66** [comment]
Yeah, thought that there is wonky stuff going on. I really like the game tho and might "steal" some of your code if that's fine. As it's really barebones, but goes in a similar direction that I want to go, I think it's a good inspiration now. So, thanks for the great work. :)

**Comment by foxnne** [comment]
Yeah! Im pretty happy with the tile based movement and animation systems, feel free to steal whatever code is applicable for you. I'm also pretty happy with the heightmap/shadow system too and my pixel art editor has a handy height map editor for making those maps. Like you said it's all bare bones but the foundation is there for the character controller and things like that.

**Comment by hardliner66** [comment]
Is this the editor you used for graphics? https://pixieditor.net/

I'm just trying to figure out how your stuff works :D

**Comment by foxnne** [comment]
> Is this the editor you used for graphics? https://pixieditor.net/
> 
> I'm just trying to figure out how your stuff works :D

So sorry I missed this and didn't get back to you. I wrote my own pixel art and animation editor I use for this project:
[pixi](https://github.com/foxnne/pixi)
